packages/design-system/src/components/CloseButton/CloseButton.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import classNames from 'classnames';
+import useId from '../utilities/useId';
+import { CloseIconThin } from '../Icons';
+
+interface BaseCloseButtonProps {
+  /**
+   * Additional classes to be added to the button element.
+   */
+  className?: string;
+  /**
+   * A custom `id` attribute for the dialog element
+   */
+  id?: string;
+}
+
+export type CloseButtonProps = Omit<
+  React.ComponentPropsWithRef<'button'>,
+  keyof BaseCloseButtonProps
+> &
+  BaseCloseButtonProps;
+
+/**
+ *
+ */
+export const CloseButton = ({ className, id: idProp, ...buttonAttributes }: CloseButtonProps) => {
+  const id = useId('close-button--', idProp);
+  return (
+    <button
+      className={classNames('ds-c-close-button', className)}
+      type="button"
+      id={id}
+      {...buttonAttributes}
```

Comment:
i feel like because this is an icon button, it needs some sort of aria-label describing what it's doing, and that label should support i18n as well

Reply:
saw aria-label was added from within other components like dialog, but i feel like there should still be a sensible fallback in case this gets used and someone doesn't realize this prop would be required

Reply:
i also don't think `ariaHidden` should be false since the button would have the label describing the action. the icon doesn't need to also provide this information - too much noise

Reply:
The aria label is coming from [this line in the dialog](https://github.com/CMSgov/design-system/pull/2828/files#diff-1d82a8d2bfc6ee2fb3dc5d5eff699b17a50835b8f9bb131ff1f87690f4f60094R159). It's an internal component, but maybe I should set the `aria-label` prop to required so we know it's being defined.

Reply:
Added in [fde5930](https://github.com/CMSgov/design-system/pull/2828/commits/fde593024df284c24c9afd41ec3a299095a12615)

Reply:
Does that alleviate your concerns, @zarahzachz?